lib/modules/pageUtils.js

```diff
@@ -32,7 +32,11 @@ export const createHash = str => {
 
 export const urlFromPage = (page, mergePage) => {
   const { queryParams, hashParams, url } = mergePage ? merge(page, mergePage) : page;
-  const queryString = Object.keys(queryParams).length ? `${createQuery(queryParams)}` : '';
-  const hashString = Object.keys(hashParams).length ? `${createHash(hashParams)}` : '';
+  const queryString = queryParams && Object.keys(queryParams).length
+    ? `${createQuery(queryParams)}`
```

Comment:
What's going on with the string substitution here. Why not just use the value that createQuery returns?